src/server.cpp

```diff
@@ -1198,6 +1198,19 @@ void Server::ProcessData(NetworkPacket *pkt)
 			return;
 		}
 
+		if (!pkt->getWasReliable() && !toServerCommandTable[command].can_be_unreliable)
+		{
+			RemotePlayer* player = m_env->getPlayer(peer_id);
+			if (player)
+				actionstream << "Recieved unreliable packet from '" << player->getName() << "'";
+			else
+				actionstream << "Recieved unreliable packet from peer_id=" << peer_id;
+			actionstream << " for command type: '" << toServerCommandTable[command].name
```

Comment:
i think actionstream is only for in-game happenings.
should be `warningstream` or `infostream`

Reply:
was thinking actionstream on the server side since it *could* indicate an attack attempt, I think for formspec submissions that gets logged to actionstream if the server hadn't sent that formspec to the client.